src/main/resources/assets/js/components/AppListComponent.jsx

```diff
@@ -99,6 +100,11 @@ var AppListComponent = React.createClass({
               </span>
             </th>
             <th className="text-right">
+              <span className={headerClassSet}>
```

Comment:
Created another issue for that here: https://github.com/mesosphere/marathon/issues/1058, thanks